apps/api/pages/api/webhooks/[id]/_patch.ts

```diff
@@ -87,6 +88,11 @@ export async function patchHandler(req: NextApiRequest) {
     args.data.userId = bodyUserId;
   }
 
+  if (args.data.eventTriggers) {
+    const uniqueEventTriggers = Array.from(new Set(args.data.eventTriggers as WebhookTriggerEvents[]));
+    args.data.eventTriggers = uniqueEventTriggers;
+  }
+
```

Comment:
```suggestion
if (args.data.eventTriggers) {
  args.data.eventTriggers = [...new Set(args.data.eventTriggers)];
}
```

We don't need the intermediary const uniqueEventTriggers here, it should be doable without the need of it. If you see any type errors, feel free to fix that within the suggestion 🙏

Reply:
Looks fine 👍